functions/APIs/queries.js

```diff
@@ -0,0 +1,95 @@
+const { db } = require('../util/admin');
+const axios = require('axios');
+require('dotenv').config();
+
+/*
+find food banks that are a specific distance from a farm
+GET /queryFoodBanksByDistance
+URL params: locationId, distance
+success response: array of objects
```

Comment:
Array of what objects?  What keys will those objects have?

Reply:
I added documentation for the properties of the objects in the POST requests.